builtins/builtins.go

```diff
@@ -840,6 +840,25 @@ func Hash(ctx context.Context, args ...object.Object) object.Object {
 	return object.NewByteSlice(h.Sum(nil))
 }
 
+func Spawn(ctx context.Context, args ...object.Object) object.Object {
+	if err := arg.Require("spawn", 1, args); err != nil {
+		return err
+	}
+	fn, ok := args[0].(*object.Function)
```

Comment:
It would be nice to also allow built-ins, don’t know if you remember from my experiment on this, those worked pretty much out of the box, the only issue was with functions and built-ins that take functions like `each`.

Reply:
Yeah, that'll definitely be included.